ex03.ex

```diff
@@ -101,7 +119,17 @@ defmodule Ex03 do
 
   """
 
-  def list_equal . . . "your code"
+  #def list_equal . . . "your code"
```

Comment:
## -2

You need to make sure your code compiles without warnings. Also
why not write

~~~ elixir
def list_equal([head1|tail1],[head2|tail2]) do
  if head1 == head2 do
    list_equal(tail1,tail2)
  else
    false
  end
end
~~~

as

~~~ elixir
def list_equal([head|tail1], [head|tail2]), do: list_equal(tail1, tail2)
def list_equal([_|_], [_|_]), do: false
~~~